packages/element-library/src/media/display.js

```diff
@@ -123,6 +124,9 @@ function MediaDisplay({
       mask={mask}
       showPlaceholder={showPlaceholder}
     >
+      {showPlaceholder &&
```

Comment:
Q: Are the `showPlaceholder` and `resourcePlaceholder` both needed for the media display?
It looks like in case of `showPlaceholder`, the following CSS is applied to media: `background-color: #C4C4C4;`. Wondering it it's still needed now?

Perhaps we could just use `renderResourcePlaceholder?.(resource)` here then.